src/app/components/shell/header/menus/menus.tsx

```diff
@@ -43,40 +43,44 @@ export default function Menus({open}: {open: string;}) {
     }, [active]);
 
     return (
-        <DropdownContextProvider>
-            <div className={cn('menus desktop', {open})}>
-                <nav className='meta-nav'>
-                    <UpperMenu />
-                </nav>
-                <nav className='nav' aria-label='Main menu'>
-                    <div className='container'>
-                        <Logo />
-                        <MainMenu />
-                    </div>
-                </nav>
-            </div>
-            <div
-                className={cn('menus mobile', {active})}
-                ref={ref}
-                onKeyDown={closeOnEsc}
-            >
-                <Logo />
-                <MenuExpander
-                    active={active}
-                    onClick={expandMenu}
-                    aria-controls='menu-popover'
-                    aria-expanded={active}
-                />
-                <div className='menu-popover-overlay' onClick={clickOverlay}>
-                    <div id='menu-popover' className='menu-popover'>
-                        <div className='menu-title'>Menu</div>
-                        <ul className='no-bullets' onKeyDown={treatSpaceOrEnterAsClick}>
-                            <MainMenuItems />
-                            <UpperMenu />
-                        </ul>
+        <React.Fragment>
```

Comment:
not sure this is needed here?

Reply:
There are two DropdownContextProvider units, so I had to wrap them in a Fragment (because of the every-component-must-return-a-single-root-element rule)